pkgs/development/ocaml-modules/junit/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildDunePackage, ptime, tyxml }:
+
+buildDunePackage (rec {
+  pname = "junit";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
```

Comment:
Please use the release tarball from https://github.com/Khady/ocaml-junit/releases/download/2.0.2/junit-2.0.2.tbz It has correct version information (you can see the difference by running `ocamlfind list`).

Reply:
Done.

Reply:
Thanks!